plugin.audio.somafm/resources/settings.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
better to convert this to the new settings format. See: https://forum.kodi.tv/showthread.php?tid=350135